src/pages/Login/LoginForm/index.tsx

```diff
@@ -0,0 +1,50 @@
+import Input from '../../../components/Input';
+import BoxForm from '../../../features/BoxForm';
+import useFormData from '../../../hooks/useFormData';
+import * as constant from './constant';
+
+export declare interface ILoginForm {
+  onSubmit: (data: IFormData) => Promise<void>
+}
+
+export interface IFormData {
+  Email: string
+  Password: string
+};
+
+export const initialState: IFormData = {
+  Email: '',
+  Password: '',
+};
+
+const LoginForm = ({ onSubmit }: ILoginForm): JSX.Element => {
+  const [formData, onValueChange, submit] = useFormData<IFormData>({ initialState, onSubmit });
+
+  return (
+    <BoxForm
+        heading={constant.FORM_HEADING}
+        buttonText={constant.SUBMIT_BUTTON_TEXT}
+        buttonOnClick={submit}
+        backButtonTo=''
+        hasBack
```

Comment:
There is no need to keep the back button on the `BoxForm` element. You can safely remove the `hasBack` parameter.
```
<BoxForm 
    heading={constant.FORM_HEADING}
    buttonText={constant.SUBMIT_BUTTON_TEXT}
    buttonOnClick={submit}
> ...
```

Reply:
Have you decided to change this?